tests/engines/registries/acceptance/overview/overview-test.ts

```diff
@@ -81,10 +81,18 @@ module('Registries | Acceptance | overview.overview', hooks => {
                 favicon: 'fakelink',
             },
         }, 'withBrand');
+        const brandName = brandedProvider.name;
+
         const reg = server.create('registration', { provider: brandedProvider });
+        const registrationTitle = reg.title;
 
         await visit(`/${reg.id}/`);
         assert.ok(document.querySelector('link[rel="icon"][href="fakelink"]'));
+
+        const expectedPageTitle = `${brandName} | ${registrationTitle}`;
+        const pageTitle = document.getElementsByTagName('title')[0].textContent;
+        assert.equal(pageTitle, expectedPageTitle);
+
```

Comment:
More of a readability preference and `assert.dom` for consistency in how we query the DOM in tests. Some cases may require the use `document.getX()`, but I think `assert.dom` might do here.
```suggestion
        assert.dom('title').hasText(`${brandProvider.name}  |  ${reg.title}`);
```

Reply:
Unfortunately, this doesn't work. I get "Element title should exist". I'm not sure why; it definitely looks like it *should* work.

Reply:
it seems it's because the `title` is outside of `#ember-testing-container` (search context). We can pass `document` as the second argument to `assert.dom()` as such `assert.dom('title', document).hasText(...)`. But given we already use `document.queryX` in these tests, this is fine as is.